lgc/include/lgc/state/PipelineState.h

```diff
@@ -269,6 +269,9 @@ class PipelineState final : public Pipeline {
   // Set the default wave size for the specified shader stage
   void setShaderDefaultWaveSize(ShaderStage stage);
 
+  // Set the wave size for the specified shader stage
+  void setShaderWaveSize(ShaderStage stage, unsigned waveSize) { m_waveSize[stage] = waveSize; }
```

Comment:
Do all wave sizes make sense? I thought that only 32 and 64 were currently allowed. If there some limitations, could we assert that nobody passes a bad wave size by accident?

Reply:
Yes, you're right. Fixed.

Reply:
Please add asserts `assert(waveSize == 32 || waveSize == 64)`

Reply:
Done.